### PR TITLE
web-ui: add refactor, add ViolationsResult component

### PR DESCRIPTION
**Pull request by rbarilani**
web-ui: add refactor, add ViolationsResult component as mentioned here: #218

### CODE REVIEW

**Comment by netme** [comment]
Please rebase to make Travis green. We did a small fix in #230

**Comment by rbarilani** [comment]
close in favor of #218

File: web-ui/src/client/app/components/violations.jsx [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+}
+
+ViolationsResult.propTypes = {
+  pending: React.PropTypes.bool,
+  complete: React.PropTypes.bool,
+  errorMsgText: React.PropTypes.string,
+  successMsgTitle:  React.PropTypes.string.isRequired,
+  successMsgText:  React.PropTypes.string.isRequired,
+  violations: React.PropTypes.array
+};
```
Comment (rbarilani) [inline web-ui/src/client/app/components/violations.jsx]: @bhaskarmelkani Please add the proptypes as I did above...

Comment (bhaskarmelkani) [inline web-ui/src/client/app/components/violations.jsx]: Done 👍

**Pull request closed without merging**